core/services/functions/listener.go

```diff
@@ -348,7 +362,37 @@ func (l *FunctionsListener) handleRequest(ctx context.Context, requestID [32]byt
 		return
 	}
 
-	computationResult, computationError, domains, err := eaClient.RunComputation(ctx, requestIDStr, l.job.Name.ValueOrZero(), subscriptionOwner.Hex(), subscriptionId, "", requestData)
+	nodeProvidedSecrets := ""
+	if l.decryptor != nil && requestData.SecretsLocation == 1 && len(requestData.Secrets) > 0 {
+		thresholdEncSecrets, userError, err := eaClient.FetchEncryptedSecrets(ctx, requestData.Secrets, requestIDStr, l.job.Name.ValueOrZero())
+
+		// To avoid a breaking change, if secrets fetching is unsuccessful,
+		// proceed by allowing the adapter handle secrets as before.
+		// Eventually, this will be deprecated and the error will be returned to the user on-chain
+		// by uncommenting the lines within the following if statements.
+		if err != nil {
+			l.logger.Errorw("failed to fetch encrypted secrets", "requestID", requestIDStr, "err", err)
```

Comment:
If it's not an error now then change to Warn or Info. Also remove the commented-out code.

Reply:
This is an error.  If the secrets are simply invalid, then the response will be a userError, however an err indicates something is wrong w/ the adapter